auth-web/src/services/business.services.ts

```diff
@@ -25,11 +25,7 @@ export default class BusinessService {
     return axios.get(`${ConfigHelper.getLegalAPIUrl()}/businesses/${businessIdentifier}`)
   }
 
-  static async createNamedBusiness (filingBody: BusinessRequest): Promise<AxiosResponse<any>> {
-    return axios.post(`${ConfigHelper.getLegalAPIUrl()}/businesses?draft=true`, filingBody)
-  }
-
-  static async createNumberedBusiness (filingBody: BusinessRequest): Promise<AxiosResponse<any>> {
+  static async createDraftFiling (filingBody: any): Promise<AxiosResponse<any>> {
```

Comment:
Does the `BusinessRequest` no longer apply? 
If the structure has changed i would create/modify the interface to suite both filings or create new ones.

Reply:
It still applicable for IA. filingBody can be BusinessRequest and RegistrationRequest and maybe more in future. Changed it to `any` to support future filing since it is only used to post a draft filing

Reply:
Same comment as above, we should modify or add to our interfaces to include some typing. 
Even if it is just a draft that is not validated. 
Even more reason to include an interface i think since we are not validating, since errors by a developer here wouldn't be caught until the point of file and pay by a client, when the validators do run.

Reply:
In our framework (TS), type checks are not done at runtime, only compile time. So yes, an interface (or partial or [union types](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types)) can avoid some errors. However, I'm going to leave it to the author to determine the best approach here at this time.

Reply:
We determined that for the time being we will enforce the base structure of the filing with the subset of the filing type optional. Will resolve!